pkg/apis/kops/cluster.go

```diff
@@ -260,7 +260,8 @@ const (
 )
 
 type LoadBalancerAccessSpec struct {
-	Type LoadBalancerType `json:"type,omitempty"`
+	Type               LoadBalancerType `json:"type,omitempty"`
+	IdleTimeoutSeconds *int64           `json:"idleTimeoutSeconds,omitempty"`
```

Comment:
`int64` _might_ be overkill here, but we can plan for the (very distant) future ;-)